tools/system_libs.py

```diff
@@ -26,7 +25,7 @@ def run_commands(commands):
   else:
     pool = shared.Building.get_multiprocessing_pool()
     # https://stackoverflow.com/questions/1408356/keyboard-interrupts-with-pythons-multiprocessing-pool, https://bugs.python.org/issue8296
-    pool.map_async(call_process, commands, chunksize=1).get(maxint)
```

Comment:
please add a comment about the constant, that it is just something huge for several days and that works in both 2 and 3